ovos_messagebus/event_handler.py

```diff
@@ -35,31 +35,45 @@ def __init__(self, application, request, **kwargs):
     def on(self, event_name, handler):
         self.emitter.on(event_name, handler)
 
+    @property
+    def filter(self) -> bool:
+        return Configuration().get("websocket", {}).get("filter", False)
+
+    @property
+    def filter_ogs(self) -> list:
+        return Configuration().get("websocket", {}).get("filter_ogs", ["gui.status.request", "gui.page.upload"])
+
     @property
     def max_message_size(self) -> int:
         return Configuration().get("websocket", {}).get("max_msg_size", 10) * 1024 * 1024
 
     def on_message(self, message):
+        if not self.filter:
```

Comment:
This could be simplified making `if self.filter` the conditional and leaving the common code here outside of the conditional

Reply:
Fair point. I put it on top because I wanted the default use case to be the first bit of code you read, rather than having to search down the method to find out what it does in a normal scenario.